selftests/functional/test_basic.py

```diff
@@ -763,6 +763,11 @@ def test_simplewarning(self):
         simplewarning.sh uses the avocado-bash-utils
         """
         os.chdir(basedir)
+        # simplewarning.sh calls "avocado" without specifying a path
+        os.environ['PATH'] += ":" + os.path.join(basedir, 'scripts')
+        # simplewarning.sh calls "avocado exec-path" which hasn't
+        # access to an installed location for the libexec scripts
```

Comment:
I don't really like the comment, how about `simplewarnings` requires avocado to be installed including the libexec scripts. Let's workaround that by adding in-git versions in PATH? Anyway this is just a minor thing so just consider that and if you like it send a fix (or let me know I can do that...)